File: Easy/168_Excel_Sheet_Column_Title.py
def convertToTitle( columnNumber):
    x = columnNumber
    ans = ""
    while (x != 0):
        num = x % 26
        if(num!=0):
            ans = chr(num+ 64) + ans
            x = x // 26
        else:
            ans = "Z" + ans
            x = (x // 26) - 1

    
    return ans


print(convertToTitle(701))

File: EasyJS/168_Excel_Sheet_Column_Title.js
var convertToTitle = function (columnNumber) {
  let ans = [];
  let temp = columnNumber;

  while (temp > 0) {
    temp--;
    let char = String.fromCharCode((temp % 26) + 65);
    ans.unshift(char);
    temp = Math.floor(temp / 26);
  }

  return ans.join("");
};
